lib/trino-collect/src/main/java/io/trino/collect/cache/NonEvictableLoadingCacheImpl.java

```diff
@@ -29,7 +29,7 @@
     public void invalidateAll()
     {
         throw new UnsupportedOperationException("invalidateAll does not invalidate ongoing loads, so a stale value may remain in the cache for ever. " +
-                "Use EvictableCacheBuilder if you need invalidation, or use SafeCaches.buildNonEvictableCacheWithWeakInvalidateAll() " +
+                "Use EvictableCache if you need invalidation, or use SafeCaches.buildNonEvictableCacheWithWeakInvalidateAll() " +
```

Comment:
`EvictableCacheBuilder` was intentionally here, since it's the class to use if you want to create an evictable cache.
Similar to `buildNonEvictableCacheWithWeakInvalidateAll` -- this is also a factory  method to get a cache

Reply:
(i don't feel strongly  about this, just sayin' this was intentional)

Reply:
I could restore it, however that makes asserting query messages complex and test will become hard to read.